api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -163,6 +163,9 @@ message Bootstrap {
   // Optional set of stats sinks.
   repeated metrics.v3.StatsSink stats_sinks = 6;
 
+  // When true, enable lazy init feature for stats that's of Stats::LazyInit<T> type.
```

Comment:
This is great; but this needs to be documented also. I think I'd put which specific stats are lazy-inited and why, in the doc.

Reply:
I meant it needs to be documented in the Envoy official documentation (rst file).